### PR TITLE
Update to the 'videos, webinars and email alerts' pattern

### PR DESCRIPTION
**Pull request by Louisehmrc**

### CODE REVIEW

**Review by Jon-Rowe-HMRC — APPROVED** [review]

File: src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+{% from "_example.njk" import example %}
+{% from "_codeSnippet.njk" import codeSnippet %}
+
+<h2 class="govuk-heading-l">When to use</h2>
+
+<p class="govuk-body">Use this pattern to design new or existing content that includes email updates, videos, webinars and community forums. It may also sometimes include e-learning and guidance.</p>
+
+<h2 class="govuk-heading-l">When not to use</h2>
+
+<p class="govuk-body">Do not use this pattern if you’re just <a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#video"> adding a video to a piece of guidance</a>.</p>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk:15]: The link should have a class of `govuk-link`

File: src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    <li>YouTube videos</li>
+    <li>upcoming webinars</li>
+    <li>recorded webinars</li>
+    <li>e-learning or guidance</li>
+    <li>HMRC community forums</li>
+</ol>
+
+<p class="govuk-body">Use descriptive level 2 headings to tell the user what they will learn.</p>
+
+<p class="govuk-body">Below the heading, describe the action users will take and what they will learn.Then link to the product.</p>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk:97]: Should be a space between `learn.` and `Then`

**Review by timsb — CHANGES_REQUESTED** [review]

File: src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk (hunk 1 above)
Comment (timsb) [reply src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk:15]: Also, I'd remove the space before the word `adding`. I don't _think_ it'll make a difference as HTML would only see add a single space but it's more semantically correct and _might_ have an effect on how it reads out to screenreaders

File: src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk (hunk 2 above)
Comment (Louisehmrc) [reply src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk:97]: Fixed.

File: src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk (hunk 1 above)
Comment (Louisehmrc) [reply src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk:15]: Fixed.

**Review by timsb — APPROVED** [review]

**Pull request merged**